packages/extension-browser/package.json

```diff
@@ -112,7 +112,7 @@
   "repository": "webhintio/hint",
   "scripts": {
     "build": "npm-run-all build:*",
-    "build-release": "npm run clean && npm run build:assets && tsc --inlineSourceMap false --removeComments true && webpack --env.release && npm run package-extension",
+    "build-release": "npm run clean && npm run build:assets && tsc --inlineSourceMap false --removeComments true && webpack --env.release",
```

Comment:
If we remove `&& npm run package-extension` here, we need to add it to the end of the `test` script or we won't get bundle size validation.